syntax = "proto2";

message example_msg {
    required uint32 cnt = 1;
}
